SpatialGDK/Documentation/content/authority.md

```diff
@@ -0,0 +1,98 @@
+# Authority in the GDK
+To work with authority in the GDK, it’s useful to refresh on authority in Unreal’s native networking.
+
+## Unreal networking authority
+
+In native-Unreal networking, the single server has absolute authority over all replicated Actors,  while clients have only proxies of Actors. Unreal models this using the `Role` and `RemoteRole` fields within an Actor.
+
+For the majority of use-cases, these fields have the values:
+
+Actor on **server**:
+
+* `Role = ROLE_Authority`
+* `RemoteRole = ROLE_SimulatedProxy`
+
+Actor on **client**:
+
+* `Role = ROLE_SimulatedProxy`
+* `RemoteRole = ROLE_Authority`
+
+Actors that have an [owning connection (Unreal documentation)](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/OwningConnections) are slightly different. An Actor can have an owning connection if:
+
+* It is a `PlayerController` with an associated `NetConnection`.
+* It is a `Pawn` which is possessed by a `PlayerController` that has an associated `NetConnection`
+* Its `Owner` is set to an Actor that has an owning connection.
+
+For example, if a client has a `PlayerController` which possesses a `Character` which is holding a gun, all three (the gun, `Character` and `PlayerController`) have an owning connection.
+
+In any of these cases, Unreal authority looks like:
+
+Actor on **server** with an **owning connection**:
+
+* `Role = ROLE_Authority`
+* `RemoteRole = ROLE_AutonomousProxy`
+
+Actor on **owning client**:
+
+* `Role = ROLE_AutonomousProxy`
+* `RemoteRole = ROLE_Authority`
+
+Actor on **non-owning client**:
+
+* `Role = ROLE_SimulatedProxy`
+* `RemoteRole = ROLE_Authority`
+
+`ROLE_AutonomousProxy` indicates that this Actor is owned by a client and can receive client RPCs called on the Actor, as well as trigger server RPCs which are handled by the server. For more information on owning connection see the [Unreal doccumentation](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/OwningConnections)
```

Comment:
```suggestion
`ROLE_AutonomousProxy` indicates that this Actor is owned by a client and can receive client RPCs called on the Actor, as well as trigger server RPCs which are handled by the server. For more information on owning connection see the [Unreal doccumentation](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/OwningConnections).
```

Reply:
How come we define this below where it's used. What about putting a description of the various Roles above and then showing how they are used on clients and servers.

Reply:
Good point